.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: c
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+sudo: required
+env:
+  - PACKAGE="tlstunnel" OCAML_VERSION=4.01   POST_INSTALL_HOOK="./.travis-test.sh"
```

Comment:
Would be better to move the `POST_INSTALL_HOOK` command into the `build-test` field of the `opam` file.